webui/src/components/RepositoryCreateForm.js

```diff
@@ -51,6 +50,7 @@ export const RepositoryCreateForm = connect(
     }, [getConfig]);
 
     const blockstoreType = config.payload ? config.payload['blockstore.type'] : DEFAULT_BLOCKSTORE_TYPE;
+    const storageNamespaceValidityRegex = new RegExp(`^(${blockstoreType}):/.*$`);
```

Comment:
I'm sure you're asking "so how come the UI is allowed to touch configuration directly?!" here.  And you're right, it shouldn't.  This is tech debt on the UI -- and there is no expectation to fix it in this PR (of course)!

Reply:
Agreed. 
But just out of curiosity, how can that be fixed? By sending only a limited set of configuration options to UI?